base_layer/p2p/tests/data/README.md

```diff
@@ -0,0 +1 @@
+Temp folder for LMDB database files
```

Comment:
Perhaps the database files should be gitignored incase they leak - I'd rename the readme to .gitkeep which is the idiomatic way to keep an 'empty' directory in the repo in git

Reply:
That is cleaner, I applied the change.